aten/src/ATen/templates/TensorBody.h

```diff
@@ -363,6 +371,15 @@ class TORCH_API Tensor {
     return !at::impl::variable_excluded_from_dispatch();
   }
 
+  inline bool is_conj() const {
+    return impl_->is_conj();
+  }
+
+  // TODO: do I need to add this method directly on the Tensor to implement conj_view() in UnaryOps?
+  inline void set_conj(bool conjugate) const {
```

Comment:
Also quite a few places rely on the invariant that the conj bit is never changed after Tensor creation (autograd saving, and is_conj() coding), so that should be mentioned in the doc here I think.

Reply:
add warning that the conj bit is supposed to be a read-only field and we only rely on this information.